presto-spark-base/src/main/java/com/facebook/presto/spark/execution/common/WorkerHiveConnectorProperties.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.spark.execution.common;
+
+import com.google.common.collect.ImmutableMap;
+
+/**
+ * Represents catalog/<connector>.properties in Presto context. It helps to create a working baseline for Prestissimo worker.
+ */
+public class WorkerHiveConnectorProperties
+        extends Properties
+{
+    public static final String CACHE_ENABLED = "cache.enabled";
+    public static final String CACHE_MAX_CACHE_SIZE = "cache.max-cache-size";
+    public static final String CONNECTOR_NAME = "connector.name";
+
+    private static final String DEFAULT_CACHE_ENABLED = "false";
+    private static final String DEFAULT_CACHE_MAX_CACHE_SIZE = "0";
+    private static final String DEFAULT_CONNECTOR_NAME = "hive";
+
+    private WorkerHiveConnectorProperties(PropertiesPopulator populator, ImmutableMap<String, String> propertiesMap)
+    {
+        super(populator, propertiesMap);
+    }
+
+    public static class WorkerHiveConnectorConfigPropertiesBuilder
+            extends Properties.PropertiesBuilder<WorkerHiveConnectorConfigPropertiesBuilder>
+    {
+        public WorkerHiveConnectorConfigPropertiesBuilder initializeDefault()
```

Comment:
Since every property implementation we always have the `initializeDefault()` method, should we add it to the base class as an abstract interface?

Reply:
We might not need to do that as we will never be using the builder interface as an interface. The things inside the interface are more for abstraction purpose.